docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/inviteToEnvelope/inviteToEnvelopeByEmail.rst

```diff
@@ -41,11 +47,80 @@ We're able to invite to an envelope such as customers of the platform and non-re
 Auto Deposit Function
 =====================
 
-The envelope sent by email can be assigned to mailbox automatically with Auto Deposit function.
+The envelope sent via email can be automatically assigned to a mailbox with the Auto Deposit function.
 
 * Open 'User Profile'
 * Select proper mailbox in 'Auto Deposit Mailbox'
 * All envelopes will go to this mailbox automatically without invitations
 * By default 'Auto Deposit Mailbox' is set to user's personal mailbox
 
-.. note:: User has an option to select auto deposit mailbox in invitation email under mailbox select menu.
\ No newline at end of file
+.. note:: The user has the option to select an auto-deposit mailbox in the invitation email under the mailbox selection menu.
+
+
+How to accept invitation and send envelope without authorisation (for guests)?
+==============================================================================
+
+1. To accept the invitation to the envelope, the user must follow the link sent to their email address. You have received a new envelope message will be displayed. 
+Click on "Agree button".
+
+.. image:: pic_inviteToEnvelopeByEmail/inviteYouRecieveEnvelope.png
+   :width: 400
+   :align: center
+
+2. The user observes limited functionality within the envelope.
+3. Only fields assigned to this user can be filled. If the next recipient's roles where assigned as same as the particular user, the user can simultaneously see recipient's fields as active for completion.
+Fill in all required fields (button "Edit" changes to "Send")
+
+.. image:: pic_inviteToEnvelopeByEmail/InviteSimpleEnvView.png
+   :width: 400
+   :align: center
+
+4. Click on "Send" button
+5. If there are roles in the envelope flow that inherit from the current role, then the envelope with the functionality of the next role (Approver/Signer) will be displayed.
+
+.. image:: pic_inviteToEnvelopeByEmail/inviteSameAsRoles.png
+   :width: 400
+   :align: center
+
+6. Perform the necessary actions depending on the role funcrionality and then complete (send) the envelope. 
+If there are no roles in the flow that inherit from the current one, then the envelope will be sent further along the flow and success page will be displayed.
+
+.. image:: pic_inviteToEnvelopeByEmail/inviteSuccessPage.png
+   :width: 400
+   :align: center
+
+
+How to accept invitation and send envelope without authorisation for registered users?
+======================================================================================
+
+1. To accept the invitation to the envelope, the user can follow the link sent to their email address. 
+Enter valid credentials and sign in. Authorization through third-party services is also available.
```

Comment:
please unify Authorization and Authorisation. Chose one pattern of the word

Reply:
done